plugins/bcc-login/src/components/send-notifications.js

```diff
@@ -9,18 +9,17 @@ const SendNotifications = ({ label, postId, status, targetGroupsCount, visibilit
     const [visible, setVisible] = useState(false);
     const toast = useRef(null);
     const [nonce, setNonce] = useState(null);
-    const [translations, setTranslations] = useState([]);
+    const [translations, setTranslations] = useState(null);
 
     useEffect(() => {
-        // WordPress exposes wpApiSettings.nonce when scripts are enqueued correctly
         const wpNonce = window?.wpApiSettings?.nonce || window?.bccLoginNonce;
         setNonce(wpNonce || null);
     }, []);
 
     const showToast = (status) => {
         const messages = {
-            success: { severity: 'success', summary: 'Success', detail: 'Varsler sendt ut!' },
-            error: { severity: 'error', summary: 'Error', detail: 'Feil ved utsending av varsler.', sticky: true },
+            success: { severity: 'success', summary: 'Suksess', detail: 'Varsler sendt ut!' },
```

Comment:
Should ideally translate these messages using WP localization - unless that hasn't been set up for this plugin